### ISSUE TITLE
Problems retrieving retsinformation.dk

### ISSUE THREAD

**Issue by saraskals** [fnielsen/dasem#9]
Hi, I'm having problems with retsinformation.py - are there any other requirements than those in the readme and requirements.txt?

**Comment by fnielsen** [comment]
Thanks for noting. I suspect there is a unmentioned dependency on `pytz`.

**Comment by fnielsen** [comment]
I have now added `pytz` to the `requirements.txt`. I hope it works now. 

Note that retsinformation.dk seems to have redesigned their website recently, so the `retsinformation.py` may no longer work appropriately. I haven't checked. :|

**Comment by saraskals** [comment]
You're right :/ The site is completely changed.. Do you know if texts from Retsinformation will be included in the Danish gigaword corpus?

**Comment by fnielsen** [comment]
It is included (at least some 5 months ago). I have just asked whether we should open the private repository.

**Comment by sonnymeyer** [comment]
@fnielsen did you get any response at that point? I noticed on the dagw page that Leon maintains, that the intention is to publish it in early 2021 (whereas it also said that in 2020) :-)

Thanks
